demos/ping_pong.py

```diff
@@ -0,0 +1,233 @@
+# 'Ping pong' interpolation experiment
+# ====================================
+#
+# In this demo, we perform a 'ping poing test', which amounts to interpolating a given
+# sensor function repeatedly between the two meshes using a particular interpolation
+# method. The purpose of this experiment is to assess the properties of different
+# interpolation methods. ::
+
+import os
+
+import matplotlib.pyplot as plt
+from firedrake import *
+from firedrake.pyplot import tricontourf, triplot
+
+from animate import *
+
+# Consider two different meshes of the unit square: mesh A, which is laid out
+# :math:`20\times25` with diagonals from top left to bottom right and mesh B
+# :math:`20\times20` with diagonals from bottom left to top right. ::
+
+mesh_A = UnitSquareMesh(20, 25, diagonal="left")
+mesh_B = UnitSquareMesh(20, 20, diagonal="right")
+
+fig, axes = plt.subplots(ncols=2, figsize=(10, 5))
+for i, mesh in enumerate((mesh_A, mesh_B)):
+    triplot(mesh, axes=axes[i])
+    axes[i].set_title(f"Mesh {'B' if i else 'A'}")
+    axes[i].axis(False)
+    axes[i].set_aspect(1)
+plt.savefig("ping_pong-meshes.jpg")
```

Comment:
I'd add `bbox_inches="tight"` to all figures here (and all current/future demos too btw).